### ISSUE TITLE
Couldn't start project on Android: Error running adb: (_Api || _load_Api(...)).default.versionsAsync is not a function

### ISSUE THREAD

**Issue by Archakov06** [expo/expo-cli#442]
I installed Android Studio, installed two emulators and ran the command `expo start --android`. But it gives this error.

![image](https://user-images.githubusercontent.com/12086860/54702789-9ae5f100-4b48-11e9-866f-3db8888009b6.png)

**Comment by SounD120** [comment]
Hi @brentvatne . Seeing the same issue when running `expo android`. Stack trace leads to `xdl` package
```
(node:13744) UnhandledPromiseRejectionWarning: TypeError: (_Api || _load_Api(...)).default.versionsAsync is not a function
    at C:\xdl@53.1.7\src\Android.js:97:28
```

**Comment by fson** [comment]
Hey all, thanks for reporting. This should be fixed in `expo-cli@2.13.0`.

**Comment by Archakov06** [comment]
@fson yes, now everything works!
